test/utils/iso8601.spec.js

```diff
@@ -55,99 +55,6 @@ describe('utils/iso8601', function() {
     }
   });
 
-  describe('#validateDate', function() {
-
-    // given
-    const VALID = [
```

Comment:
Where did these tests go? Moved elsewhere?

Reply:
Gone as duplicates of https://github.com/camunda/bpmnlint-plugin-camunda-compat/blob/2180dd35df28b48de6e9773046d69abdbe6e4fdb/test/utils/iso8601.spec.js#L12